flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -1121,6 +1122,27 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     expectedProperty.put("k2", "b")
     assertEquals(expectedProperty, database.getProperties)
```

Comment:
`CatalogTableITCase` is mostly for testing the interaction with the catalog. Can we move this test to `CalcITCase`?